src/openrct2/Diagnostic.h

```diff
@@ -70,20 +70,20 @@ enum class DiagnosticLevel
 
 extern bool _log_levels[static_cast<uint8_t>(DiagnosticLevel::Count)];
 
-void diagnostic_log(DiagnosticLevel diagnosticLevel, const char* format, ...);
-void diagnostic_log_with_location(
+void DiagnosticLog(DiagnosticLevel diagnosticLevel, const char* format, ...);
+void DiagnosticLogWithLocation(
     DiagnosticLevel diagnosticLevel, const char* file, const char* function, int32_t line, const char* format, ...);
 
 #ifdef _MSC_VER
 #    define diagnostic_log_macro(level, format, ...)                                                                           \
-        diagnostic_log_with_location(level, __FILE__, __FUNCTION__, __LINE__, format, ##__VA_ARGS__)
+        DiagnosticLogWithLocation(level, __FILE__, __FUNCTION__, __LINE__, format, ##__VA_ARGS__)
 #else
 #    define diagnostic_log_macro(level, format, ...)                                                                           \
-        diagnostic_log_with_location(level, __FILE__, __func__, __LINE__, format, ##__VA_ARGS__)
+        DiagnosticLogWithLocation(level, __FILE__, __func__, __LINE__, format, ##__VA_ARGS__)
 #endif // _MSC_VER
 
 #define log_fatal(format, ...) diagnostic_log_macro(DiagnosticLevel::Fatal, format, ##__VA_ARGS__)
```

Comment:
You missed these defines, which for all practical purposes are function names too.

Reply:
I'll add those too.